Source/WebKit/GPUProcess/graphics/WebGPU/RemoteGPU.cpp

```diff
@@ -58,13 +58,15 @@ RemoteGPU::~RemoteGPU() = default;
 
 void RemoteGPU::initialize()
 {
+    assertIsMainRunLoop();
```

Comment:
EWS reported this assertion failed.
https://gist.github.com/fujii/2bcbddabb88ae95b2a0108e849f85c03